test/shortest_path.jl

```diff
@@ -0,0 +1,32 @@
+using Graphs
+using GraphsOptim
+using Test
+
+num_vertices = 20
+
+function test_shortest_path(g)
+    edge_cost = rand(1:100, num_vertices, num_vertices) .* adjacency_matrix(g)
```

Comment:
better use fixed graphs for which we know the answer

Reply:
Randomness removed and now uses shortest path example on Wikipedia for testing